### PR TITLE
chore: Minor refactoring for consistency

### PR DESCRIPTION
**Pull request by lipis**
- `getNormalizedZoom` -> `normalizeZoomValue `
- `roomID` -> `roomId`
- various `i` -> `index`
- Added autofixable rule to ESLint: `object-shorthand`
- Added autofixable rule to ESLint: `no-var`

### CODE REVIEW

**Comment by dwelle** [comment]
We've already went once from `roomId` → `roomID` and it ended up breaking clients due to the SW caching issue (you recently asked about it in a different PR), so let's not a do a repeat please :).

**Comment by lipis** [comment]
Goldfish memory :) But still.. would it be a real problem? We can merge on sunday morning to have as few failures as possible (if any)..

**Comment by dwelle** [comment]
I'm afraid that time of the day or day of the week won't change it much :).

File: src/components/App.tsx [hunk 1]
```diff
@@ -962,7 +962,7 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         zoom: this.state.zoom,
         remotePointerViewportCoords: pointerViewportCoords,
         remotePointerButton: cursorButton,
-        remoteSelectedElementIds: remoteSelectedElementIds,
```
Comment (dwelle) [inline src/components/App.tsx:965]: not a fan of the eslint rule because we'll end up with less readable cases like this

Comment (lipis) [reply src/components/App.tsx:965]: I'll disagree with the "less readable" as you have to read less, and we were already using this rule in various places in the codebase.. this just makes it autofixable and consistent..

Comment (dwelle) [reply src/components/App.tsx:965]: Anyway, let's remove the `roomId` change and we can merge

**Comment by dwelle** [comment]
Ok, I was wrong about the `roomId` change as I mistook with `socketId`. Sorry about that.

**Pull request merged**